config/objects/generic.json

```diff
@@ -970,7 +970,7 @@
 	"favorableWinds" : {
 		"index" :225,
 		"handler": "terrain",
-		"types" : { "object" : { "index" : 0, "battleground": "favorable_winds" } }
+		"types" : { "object" : { "index" : 0, "battleground": "ship" } }
```

Comment:
Can favourableWinds be placed not on the water?

Reply:
Not in original map editor